src/NuGet.Core/NuGet.Protocol/Plugins/PluginFactory.cs

```diff
@@ -165,7 +165,10 @@ public void Dispose()
 #else
             var startInfo = new ProcessStartInfo
             {
-                FileName = Environment.GetEnvironmentVariable("DOTNET_HOST_PATH"),
+                FileName = Environment.GetEnvironmentVariable("DOTNET_HOST_PATH") ??
```

Comment:
Compile error:   Plugins\PluginFactory.cs(169,29): error CS0103: The name 'RuntimeEnvironmentHelper' does not exist in the current context [E:\A\_work\290\s\src\NuGet.Core\NuGet.Protocol\NuGet.Protocol.csproj]

Reply:
My bad... I didn't want to build the whole stuff (configure.ps1/build.ps1...). 

I used the full name  `NuGet.Common.RuntimeEnvironmentHelper.IsWindows` to avoid a `using NuGet.Common` that would be useless when `IS_DESKTOP` (and would have need a ugly `#if !IS_DESKTOP` conditional). 

I eventually ran the tests locally. The tests fail but somewhere else...

![image](https://user-images.githubusercontent.com/1583378/78897118-f9428200-7a71-11ea-8e26-561b06bdad6e.png)

I'm sorry I wasted your time.

Reply:
No worries. I'll rerun again, I'm not expecting those tests to be in any way related to this change.